Formula/qemu.rb

```diff
@@ -2,23 +2,16 @@ class Qemu < Formula
   desc "Emulator for x86 and PowerPC"
   homepage "https://www.qemu.org/"
   license "GPL-2.0-only"
-  revision 1
   head "https://git.qemu.org/git/qemu.git"
 
   stable do
-    url "https://download.qemu.org/qemu-5.2.0.tar.xz"
-    sha256 "cb18d889b628fbe637672b0326789d9b0e3b8027e0445b936537c78549df17bc"
+    url "https://download.qemu.org/qemu-6.0.0.tar.xz"
+    sha256 "87bc1a471ca24b97e7005711066007d443423d19aacda3d442558ae032fa30b9"
 
-    # remove in next release
+    # fix xcode search version header error
     patch do
-      url "https://git.qemu.org/?p=qemu.git;a=patch;h=0dbce6efb5ff2e7113734d3a0cabbf87fc56feec"
-      sha256 "8ced33c7f829216544b762d8db0f143dbea04fa5a1ce41b491bbd7808f64a944"
-    end
-
-    # remove in next release
-    patch do
-      url "https://git.qemu.org/?p=qemu.git;a=patch;h=cb7abd8319d19000b57ae6c5c474c2635db054c6"
-      sha256 "818ad42f0cb25ab5df37058e27d7f879e4389489f692da4404c1f15dde5b2c4d"
+      url "https://gist.githubusercontent.com/tylinux/1e12e733728413ffa8bc7e7a1eab437c/raw/89f7efa1ec8a62d634bd6856ffc3026dd1d06447/qemu_macos_11.3.patch"
```

Comment:
This will need to be upstreamed

Reply:
This seems to be an issue on systems with case insensitiv filesystems and libc++ 8 and newer.
Apparently FreeBSD had similar issues in the ports tree around March 2019 (like https://github.com/freebsd/freebsd-ports/commit/65232cf5a26581b4c1e83e2e48bb93641dc050d1 and many more). 
Apple upgraded LLVM from 10.0.0 to 11.1.0. Not sure if this is a libc++ regression, a macOS specific bug or intended C++20 behaviour.

Reply:
The issue comea from having files called `version` (or perhaps `VERSION`) in your include paths since some headers will now do `#include <version>`.

I suspect putting files that aren't headers in your include paths is just not a good idea. At the very least, if you have to, don't give them generic names that could easily be in someone's `#include` directive.

Reply:
Apple only added the `version` C++20 header in the just-released Xcode 12.5 which is why it's appearing now.

This is a general incompatibility with the C++20 standard library and thus should be in upstream's interest to accept such a patch.

Reply:
Thanks for the explanation. I reported the issue here: https://gitlab.com/qemu-project/qemu/-/issues/129.